futurenhs.api/FutureNHS.Api/DataAccess/Database/Read/GroupDataProvider.cs

```diff
@@ -208,27 +208,27 @@ SELECT COUNT(*)
         public async Task<Group?> GetGroupAsync(string slug, Guid userId, CancellationToken cancellationToken = default)
         {
             const string query =
-                @"SELECT DISTINCT g.Id AS Id, g.ThemeId AS ThemeId, g.Slug AS Slug, g.Name AS Name, g.Subtitle AS Strapline, g.PublicGroup AS IsPublic,( SELECT      CASE 
-                                                                                    WHEN        groupUser.MembershipUser_Id = @UserId
```

Comment:
would `WHEN        groupUser.MembershipUser_Id = @UserId` always be true because of the match on `LEFT JOIN GroupUser gu ON (gu.Group_Id = g.Id and gu.MembershipUser_Id = @UserId)`

Reply:
yes, thanks mate